tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -18,3 +18,16 @@ let ``dateTime creates System.DateTime instances`` count =
     |> List.distinct
     |> List.length
     =! actual.Length
+
+[<Fact>]
+let ``unicode doesn't return any surrogate`` () =
+    let actual = Gen.sample 100 100000 Gen.unicode 
+    [] =! List.filter System.Char.IsSurrogate actual
+
+[<Theory>]
+[<InlineData(65534)>]
+[<InlineData(65535)>]
+let ``unicode doesn't return any noncharacter`` nonchar =
+    let isNoncharacter = (=) <| Operators.char nonchar
+    let actual = Gen.sample 100 100000 Gen.unicode
+    [] =! List.filter isNoncharacter actual
```

Comment:
@moodmosaic There is a deterministic approach to test this function? These two tests are a little slow. The test run time on my computer is about twice as slow as before.  Is it OK? or it's better to remove the tests?